reactnative/ReactNativeTemplateApp/build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: 'android'
 
 dependencies {
-    compile project(':libs:SmartSync')
+    compile project(':libs:SalesforceSDK')
```

Comment:
Your build.gradle tweaking code (which puts in the :forcedroid:libs:xxx dependency) only looks for libs:SalesforceSDK.